kubeflow/kpt-set.sh

```diff
@@ -1,22 +1,58 @@
-
-# Use kpt to set kustomize values
+# Set kpt setters using env vars.
 
 set -ex
 
-kpt cfg set -R .  gke.private false
-kpt cfg set -R .  asm-label ${ASM_LABEL}
+if [[ -z "${KF_NAME}" ]]; then
+    echo "KF_NAME env var is required"
```

Comment:
Should we `exit 1` when user doesn't set required env variable?

Reply:
Good point, forgot that!